ui/webui/resources/BUILD.gn

```diff
@@ -164,6 +168,11 @@ copy("static_copy") {
   outputs = [ "$brave_resources_extra_grdps_path/{{source_target_relative}}" ]
 }
 
+copy("leo_static_copy") {
+  sources = rebase_path(leo_static_files, ".", "../../../node_modules/@brave/leo/tokens")
+  outputs = [ "$brave_resources_extra_grdps_path/css/{{source_file_part}}" ]
```

Comment:
nit: repeating css/ prefix, how about just using `{{source}}` since it's already relative?

Reply:
Unfortunately, if I do that it will put the file at `../../../node_modules/@brave/leo/tokens`